openslides_backend/services/datastore/extended_adapter.py

```diff
@@ -372,6 +372,8 @@ def _filter_changed_models(
                 formatted_args.append(
                     f'self._comparable(model.get("{arguments[i]}"), {val_str}) and model.get("{arguments[i]}")'
                 )
+            elif match[0] == "ilike":
+                raise NotImplementedError("%= is not implemented for changed models")
```

Comment:
out of scope for this PR. I don't know if we even need this feature, so I'll suppose we let it stay unimplemented for now.